lib/generators/templates/component.es6.tsx

```diff
@@ -0,0 +1,24 @@
+<%= file_header %>
+interface I<%= component_name %>Props {
+<% if attributes.size > 0 -%>
+<% attributes.each do | attribute | -%>
+<% if attribute[:union] -%>
+  <%= attribute[:name].camelize(:lower) %>: <%= attribute[:name].titleize %>;
+<% else -%>
+  <%= attribute[:name].camelize(:lower) %>: <%= attribute[:type] %>;
+<% end -%>
+<% end -%>
+<% end -%>
+}
```

Comment:
I think It would be good to use indentation here.

Reply:
@ahangarha 
Do you mean this?

```erb
interface I<%= component_name %>Props {
  <% if attributes.size > 0 -%>
  <% attributes.each do | attribute | -%>
  <% if attribute[:union] -%>
  <%= attribute[:name].camelize(:lower) %>: <%= attribute[:name].titleize %>;
  <% else -%>
  <%= attribute[:name].camelize(:lower) %>: <%= attribute[:type] %>;
  <% end -%>
  <% end -%>
  <% end -%>
}
```

But All `<%` in template have no indentation.

Reply:
I can say the indentations there are not ideal. Better to make a little improvement in each PR.

Reply:
@ahangarha 
OK. I have tried fixing indentation.

Reply:
I think it can still be improved, but also the issue is beyond the changes in this particular PR.

So, I don't want to block the PR for this specific reason.

@Judahmeek If there is no other issue except this one, I think we are good to move forward.